### PR TITLE
Eodhp 318 argo workflow scheduling

### PR DESCRIPTION
**Pull request by SerkanDural**
Updates requested by https://telespazio-uk.atlassian.net/browse/EODHP-318 are done within the scope of this PR.

urlnames.json file is created which includes list of url names to be tested.
a configmap is created using urlnames.json file and then the configmap is mounted as input file to cronworkflow.
Argo cronworkflow is created and execution is verified using Argo Cron Workflow Web UI.

Deployment of Argo Cron workflow is added to argocd kustomization file (which is within the scope of another PR in eodhp-argocd-deployment). Deployment and argocd implaementation of configmap is scope of another EODHP Story.359)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,4 +94,46 @@ Create workflow which includes reference to workflowtemplate as usual
...
+`argo list -n argo`
+
+then delete the one selected
+`argo delete <name of the workflow> -n argo`
+
+
+## creating configmap with json file content
+`kubectl create configmap urlnames-configmap --from-file=urlnames.json -n argo`
 
+the `urlnames-configmap` can then be used as vlume in argo workflow
```
Comment (hcollingwood) [inline README.md:139]:
```suggestion
the `urlnames-configmap` can then be used as volume in argo workflow
```

File: system-tests/homepage-test-fromfile-cron.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            countSuccess = 0
+            countFail = 0
+
+            try:
+
+              with open('/mnt/input/keys') as file:
+                filecontents = file.read()
+                data = json.loads(filecontents)
+                for item in data: # v in data.values():
+                  # print("the url name read from json file is {}, now cheeecking if url is up and running".format(item["url"]))
```
Comment (hcollingwood) [inline system-tests/homepage-test-fromfile-cron.yaml:34]:
```suggestion
                  # print("the url name read from json file is {}, now checking if url is up and running".format(item["url"]))
```

File: README.md [hunk 3]
```diff
@@ -67,6 +67,7 @@ where, the urlfile.yaml is
 `url: https://dev.eodatahub.org.uk`
 
 
+
```
Comment (hcollingwood) [inline README.md:70]: Consider removing new line?

Comment (SerkanDural) [reply README.md:70]: updated in new commit

**Review by hcollingwood — APPROVED** [review]

**Pull request merged**